code/modules/mob/living/carbon/resist.dm

```diff
@@ -107,7 +107,7 @@
 	if(HULK in mutations)
 		return 1
 
-	if (get_species() == "Unathi" && gender == "male")
+	if(species.breakcuffs)

```

Comment:
Ya, this basically makes gender based cuff breaking impossible. You could have species.breakcuffs as a list()? And go `if (src.gender in src.species.breakcuffs)`? Alternatively. Make `/datum/species/can_break_cuffs(var/mob/living/carbon/human/H)` which can enable you to modify each species' condition for cuff breaking?